File: NTech.Shared/Models/ResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static System.Runtime.InteropServices.JavaScript.JSType;

namespace NTech.Shared.Models
{
    public class ResponseModel
    {
        public ResponseModel()
        {
            show_error = true;
            is_error = false;
            error_list = [];
            show_success = false;
            success_message = "Completed Successfully";
            is_exception = false;
            error_title = "An error has occured";
            show_exception = true;
        }

        public object? model { get; set; }
        public bool? show_error { get; set; }
        public bool? is_error { get; set; }
        public List<string>? error_list { get; set; }
        public string? error_title { get; set; }
        public bool? show_success { get; set; }
        public string? success_message { get; set; }
        public bool? is_exception { get; set; }
        public bool? show_exception { get; set; }
    }
}

File: NTech.WebApp/db/models/response_model.ts
export class ResponseModel {
  public model: any = Object.assign(new Object());
  public error_list: Array<string> = new Array<string>();
  public show_error: boolean | null = true;
  public is_error: boolean | null = false;
  public error_title: string | null = "";
  public show_success: boolean | null = true;
  public success_message: string | null = "";
  public is_exception: boolean | null = false;
  public show_exception: boolean | null = true;
}
